python/composio/cli/execute.py

```diff
@@ -0,0 +1,46 @@
+import json
+import typing as t
+
+import click
+
+from composio.cli.context import Context, pass_context
+from composio.client.enums import Action
+from composio.exceptions import ComposioSDKError
+
+
+@click.command(name="execute")
```

Comment:
Any reason for this to be separate from than actions command?
Better to have it in the action umbrella?